### PR TITLE
Make the timeline less noisy for screen readers

### PR DESCRIPTION
**Pull request by turt2live**
There's some room for improvement here: we could be making the reader say "You sent a message at 18:46  [body]" or "TravisR sent a message at 18:46 [body]" to make it clearer. Currently it'll read out the sender profile if it is available, and if not just the timestamp + message. This improvement is best left to a future PR though.

Fixes https://github.com/vector-im/riot-web/issues/2716
Fixes https://github.com/vector-im/riot-web/issues/5697
See https://github.com/vector-im/riot-web/issues/9747

### CODE REVIEW

File: src/components/views/rooms/EventTile.js [hunk 1]
```diff
@@ -630,8 +636,12 @@ module.exports = withMatrixClient(React.createClass({
             onFocusChange={this.onActionBarFocusChange}
         /> : undefined;
 
-        const timestamp = this.props.mxEvent.getTs() ?
-            <MessageTimestamp showTwelveHour={this.props.isTwelveHour} ts={this.props.mxEvent.getTs()} /> : null;
+        const timestamp = this.props.mxEvent.getTs()
+            ? <MessageTimestamp
+                showTwelveHour={this.props.isTwelveHour}
+                ts={this.props.mxEvent.getTs()}
+                ariaHidden={muteScreenReader}
```
Comment (jryans) [inline src/components/views/rooms/EventTile.js]: Hmm, if it's muted when sending or sending is complete, when will it be read...?

Comment (turt2live) [reply src/components/views/rooms/EventTile.js]: The life of an EventTile is 3 parts, where the third is when things are read out. The first stage is the sending state, which is easy to filter. When the event switches to sent, the tile unmounts and a new one is put in place with a null sending state. That state is replaced shortly after with a property update, which triggers a re-render where the screen reader picks up the text to read (the div is replaced).

Comment (jryans) [reply src/components/views/rooms/EventTile.js]: Just for my own understanding, what does "that state is replaced shortly after with a property update" mean? Which property updates...?

Comment (turt2live) [reply src/components/views/rooms/EventTile.js]: The mxEvent changes in the component's properties, which leads to an eventSendState change of null->sent

Comment (jryans) [reply src/components/views/rooms/EventTile.js]: Huh, I thought `sent` was first (while waiting for remote echo), then `null` was the last and final status...

Comment (turt2live) [reply src/components/views/rooms/EventTile.js]: apparently it's just not my day - you're right. Will open another PR to flip it around.

**Pull request merged**